init.lua

```diff
@@ -534,15 +534,15 @@ travelnet.add_target = function( station_name, network_name, pos, player_name, m
    elseif( not( minetest.check_player_privs(player_name, {interact=true}))) then
 
       travelnet.show_message( pos, player_name, S("Error"),
-	S("There is no player with interact privilege named '%s'. Aborting."):format(tostring( player_name )));
+	S("Access has been denied. There is no player with interact privilege named '%s'."):format(tostring( player_name )));
```

Comment:
```suggestion
	S("Access denied. There is no player with interact privilege named '%s'."):format(tostring( player_name )));
```

Reply:
Again, "Access has been denied" is a sentence. Sentences are what I'm going for here. Sentences are more clear than interjections in some cases, both at first glance and for international use. Consider "Access is denied," "Travelnet denied access," or "You are not allowed," or some other sentence.

Reply:
I improved it further in the latest change (see full changes list), so it says "Access is denied" now.

Reply:
Well, I thought it was correct, but it looks good.
Sorry for wasting your time.